docs/changelog.rst

```diff
@@ -7,6 +7,9 @@ Changelog
 -------------------
```

Comment:
Could you set the date to march 2025 we then we do a release after this is merged?